lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -6,31 +6,38 @@ cache:
   paths:
     - .gradle/
 
-.debug_rules: &debug_rules
-  - if: $CI_COMMIT_TAG
-    when: never # Never build debug builds for tags
-  - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
-  - if: $CI_PIPELINE_SOURCE == "merge_request_event"
-  - if: $CI_PIPELINE_SOURCE == "web"
-
-.release_rules: &release_rules
-  - if: $CI_COMMIT_TAG
-
 variables:
   IMAGE_PREFIX: ""
   DEFAULT_IMAGE: "jangrewe/gitlab-ci-android"
   DEPLOY_DEBUG_APK_SLACK_MESSAGE: "Hello Team! Here is the latest debug APK from branch `${CI_COMMIT_REF_NAME}`. It was triggered by: `${CI_PIPELINE_SOURCE}`."
-  DEPLOY_DEBUG_APK_PATH: app/build/outputs/apk/debug
-  DEPLOY_DEBUG_APK_NAME: yourdebugapkname
+  DEPLOY_DEBUG_APK_PATH: app/build/outputs/apk
+  DEPLOY_DEBUG_APK_NAMES: yourdebugapkname
   DEPLOY_RELEASE_APK_SLACK_MESSAGE: "Hello Team! Here is the latest release APK triggered by tag: `${CI_COMMIT_TAG}`"
-  DEPLOY_RELEASE_APK_PATH: app/build/outputs/apk/release
-  DEPLOY_RELEASE_APK_NAME: yourreleaseapkname
+  DEPLOY_RELEASE_APK_PATH: app/build/outputs/apk
+  DEPLOY_RELEASE_APK_NAMES: yourreleaseapkname
```

Comment:
Replacing the old variable with this one breaks the 'interface', so will this be a new release version?

Reply:
Good point. Will merge this into release/2.x.x since breaking change.